Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -132,8 +132,8 @@ - (instancetype)initWithArticleTitle:(MWKTitle*)title
 
 - (void)showLanguagePicker {
     LanguagesViewController* languagesVC = [LanguagesViewController wmf_initialViewControllerFromClassStoryboard];
-    languagesVC.downloadLanguagesForCurrentArticle = YES;
-    languagesVC.languageSelectionDelegate          = self;
+    languagesVC.articleTitle              = self.articleTitle;
```

Comment:
Already boycotted a few things… I feel good enough. Lets get a beta out instead of refactoring.